pkg/docker/docker_host.go

```diff
@@ -44,17 +44,8 @@ func NewDockerHosts(endpoints []string, localHostIP, dockerIntf string) ([]*Dock
 }
 
 func newDockerHost(endpoint, hostIP, intf string) (*DockerHost, error) {
-	path := os.Getenv("DOCKER_CERT_PATH")
-	var err error
-	var client *dc.Client
-	if len(path) > 0 {
-		ca := fmt.Sprintf("%s/ca.pem", path)
-		cert := fmt.Sprintf("%s/cert.pem", path)
-		key := fmt.Sprintf("%s/key.pem", path)
-		client, err = dc.NewTLSClient(endpoint, cert, key, ca)
-	} else {
-		client, err = dc.NewClient(endpoint)
-	}
```

Comment:
What about this?

Reply:
It's all covered, dude. That's the whole idea. It now handles both modes.